mmdet/core/mask/mask_target.py

```diff
@@ -18,6 +18,26 @@ def mask_target(pos_proposals_list, pos_assigned_gt_inds_list, gt_masks_list,
 
     Returns:
         list[Tensor]: Mask target of each image.
+
+    Example:
+        >>> from mmdet.core.mask.mask_target import *  # NOQA
+        >>> import mmdet
+        >>> import mmcv
+        >>> from mmdet.core.mask import BitmapMasks
+        >>> H, W = 14, 14
+        >>> cfg = mmcv.Config({'mask_size': 14})
+        >>> pos_proposals_list = [torch.Tensor([
+        >>>     [176.2425,   5.5929, 190.9414, 204.9541],
+        >>>     [ 57.3241, 133.6170, 197.3850, 195.3102],
+        >>>     [ 74.8448, 186.4010, 176.0314, 203.7681],
+        >>>     [ 25.9790,  32.6989, 122.4416,  81.8580],
+        >>>     [  0.0000,   0.0000, 120.1398,  79.8232],
+        >>> ])]
+        >>> pos_assigned_gt_inds_list = [torch.Tensor([0, 1, 2, 1, 1]).long()]
+        >>> rng = np.random.RandomState(0)
+        >>> gt_masks_list = [BitmapMasks((rng.rand(3, H, W)), height=H, width=W)]
+        >>> mask_targets = mask_target(
+        >>>     pos_proposals_list, pos_assigned_gt_inds_list, gt_masks_list, cfg)
```

Comment:
We may validate the shape of mask_targets?

Reply:
It was a good idea to do this, because the way I had the test written it was slightly ambiguous what the shape was supposed to be. It should now be much clearer.